doc/source/development/release_process.rst

```diff
@@ -0,0 +1,247 @@
+
+    I propose the following artifacts for release as <version>.
+
+    sha1: <git-sha>
+
+    Git: http://git-wip-us.apache.org/repos/asf?p=cassandra.git;a=shortlog;h=refs/tags/<version>-tentative
+
+    Artifacts: https://repository.apache.org/content/repositories/orgapachecassandra-<nexus-id>/org/apache/cassandra/apache-cassandra/<version>/
+
+    Staging repository: https://repository.apache.org/content/repositories/orgapachecassandra-<nexus-id>/
+
+    The distribution packages are available here: https://dist.apache.org/repos/dist/dev/cassandra/${version}/
+
+    The vote will be open for 72 hours (longer if needed).
+
+    [1]: (CHANGES.txt) https://git1-us-west.apache.org/repos/asf?p=cassandra.git;a=blob_plain;f=CHANGES.txt;hb=<version>-tentative
+    [2]: (NEWS.txt) https://git1-us-west.apache.org/repos/asf?p=cassandra.git;a=blob_plain;f=NEWS.txt;hb=<version>-tentative
+
+
+
+Post-vote operations
+====================
+
+Any PMC can perform the following steps to formalize and publish a successfully voted release.
+
+Publish Artifacts
+-----------------
+
+Run the following commands to publish the voted release artifacts::
+
+    cd ~/git
+    git clone https://github.com/apache/cassandra-builds.git
+    # edit the variables at the top of `finish_release.sh`
+
+    # After cloning cassandra-builds repo, `finish_release.sh` is run from the actual cassandra git checkout,
+    # on the tentative release tag that we wish to tag for the final release version number tag.
+    cd ~/git/cassandra/
+    git checkout <version>-tentative
+    ../cassandra-builds/cassandra-release/finish_release.sh -v <version> <staging_number>
+
+If successful, take note of the email text output which can be used in the next section "Send Release Announcement".
+The output will also list the next steps that are required. The first of these is to commit changes made to your https://dist.apache.org/repos/dist/release/cassandra/ checkout.
+
+
+Promote Nexus Repository
+------------------------
+
+ * Login to `Nexus repository <https://repository.apache.org>`_ again.
+ * Click on "Staging" and then on the repository with id "cassandra-staging".
+ * Find your closed staging repository, right click on it and choose "Promote".
+ * Select the "Releases" repository and click "Promote".
+ * Next click on "Repositories", select the "Releases" repository and validate that your artifacts exist as you expect them.
+
+Sign and Upload Distribution Packages to Bintray
+---------------------------------------
+
+Run the following command::
+
+    cd ~/git
+    svn mv https://dist.apache.org/repos/dist/dev/cassandra/<version> https://dist.apache.org/repos/dist/release/cassandra/
+    svn co https://dist.apache.org/repos/dist/release/cassandra/<version> cassandra-dist-<version>
+    cd cassandra-dist-<version>
+
+    createrepo .
+    gpg --detach-sign --armor repodata/repomd.xml
+    for f in `find repodata/ -name *.bz2`; do
+      gpg --detach-sign --armor $f;
+    done
```

Comment:
@mshuler  where are these lines supposed to be run from? (i still quite confused :)

Reply:
@mshuler ping

Reply:
`createrepo` and the gpg signatures are for the RPM repository, so these are run within the `redhat/XYx/` directory for the relevant release version that is being worked on to a) create the yum metadata, b) sign the metadata, and c) sign some files within the signed repo metadata that the ASF sig tool errors out on.